src/impl/unicast/transport/NServiceBus.Unicast.Transport.Transactional.Config/Bootstrapper.cs

```diff
@@ -51,5 +51,6 @@ public void Init()
         public static IsolationLevel IsolationLevel { get; set; }
         public static TimeSpan TransactionTimeout { get; set; }
         public static bool SupressDTC { get; set; }
+        public static bool SuppressOutdatedTimeoutDispatchWarning { get; set; }

```

Comment:
true, i considered having everything in one place more beneficial since we only use the bootstrapper internally anyway.